.orchestra/ci/install-system-dependencies.sh

```diff
@@ -41,6 +41,7 @@ apt-get -qq install --no-install-recommends --yes \
   flex \
   g++-multilib \
   gawk \
+  gettext \
```

Comment:
We're going to merge soon a branch where for each dep you need to say why it's needed.
Who needs this?

Reply:
glibc wants `msgfmt` to build